assets/src/edit-story/theme.js

```diff
@@ -76,6 +65,37 @@ export const GlobalStyle = createGlobalStyle`
   }
 `;
 
+const GlobalObtrusiveScrollbarStyle = createGlobalStyle`
+  /*
+   * Custom dark scrollbars for Chromium & Firefox.
+   * Scoped to <Editor> to make sure we don't mess with WP dialogs
+   * like the Backbone Media Gallery dialog.
+   */
+  * {
+    scrollbar-width: thin;
+    scrollbar-color: ${({ theme }) => theme.colors.bg.v10}
+    ${({ theme }) => rgba(theme.colors.bg.v0, 0.1)};
+  }
+
+  *::-webkit-scrollbar {
+    width: ${SCROLLBAR_WIDTH}px;
+    height: ${SCROLLBAR_WIDTH}px;
+    position:fixed;
+  }
```

Comment:
So, if you try to interact with this nice custom scrollbar, you get a really ugly one. But maybe that's unavoidable?

![weird-scrollbar](https://user-images.githubusercontent.com/637548/87742712-4b075080-c7b5-11ea-8726-5203b644182e.gif)

Reply:
It seems so, at least I couldn't find a way how to influence the style -- it's partially transparent, the scrollbar thumb, meaning that it gets quite dark with our background.

Reply:
So our custom styles are not applied anymore? Any way to debug this? 🤔

Reply:
By removing the styles from global styles, now the default style is applied for the auto-hiding scrollbars which seems to be what's seen on the gif above. Only when adding back the style to global our style is applied again but then the scrollbars are always visible as well again.

One option could be trying to use the `browserHasObtrusiveScrollbars` for detecting the OS configuration but still using hovering to display the otherwise hidden scrollbars as suggested in https://github.com/google/web-stories-wp/pull/3167 Not sure if that would work as expected.

Thoughts?

Reply:
I actually only now realize, that the scrollbars I'm seeing is not in the above video. It's apparently not captured by either screenshots or screengrabs. Weird stuff. I'm seeing an os-native scrollbar which is really ugly, as it looks very weird when it was a nice dark one there, when I tried to interact with it.

This video actually looks nothing like what the scrollbar looks like on my screen. I get a nice narrow scrollbar, not this wide one in the video. And when hovered (while visible), it turns into the OS native one. 🤷

Reply:
Hmm? Do I understand correctly that you're seeing scrollbars that are not visible on screenshots? 😕 That's odd.

Does this happen when the scrollbars are always on? Or when they are auto-hiding? Did you refresh the page after changing the OS setting (in case you did change these while testing)?

I do see some "ugly" scrollbars when I change the OS settings but only if I don't refresh the page.

Reply:
@barklund Also, is this with Chrome? Or with all browsers? Can you take a photo of the screen with your phone? Or perhaps even the phone doesn't capture these ugly magic scrollbars? 😛

Reply:
Actual magic scrollbar:

![scrollbar](https://user-images.githubusercontent.com/637548/87963618-fdc20200-ca86-11ea-99a4-af16892a0d06.gif)

Reply:
@samitron7 It looks like using the auto-hiding scrollbar is also using a default style and our custom styling seems to be ignored in this case.

The auto-hiding scrollbar looks generally like this:
![scrollin1](https://user-images.githubusercontent.com/3294597/87976328-3a185100-cacd-11ea-9546-7b86bccc8b0a.gif)

However, when hovering over the scrollbar while scrolling, it looks like this:
![scrolling2](https://user-images.githubusercontent.com/3294597/87976630-bd39a700-cacd-11ea-9796-20fcdb81cc4d.gif)

The default auto-hiding scrollbar has some transparency so it looks a bit different depending on the background color. For example, GitHub is also using the same scrolling and here it looks a bit lighter due to the white background.

Is this default styling acceptable from UX perspective?

Reply:
Argghh @miina the default auto-hiding is a bit of an eyesore. Scrolling is going to be occurring all the time so I'm hoping we can fix this. @pbakaus do you have ideas how we can easily fix this

Reply:
I will test out another approach then -- the auto-hiding transition might not be that smooth but at least it should respect the custom styles (if it works).

Reply:
@samitron7 One alternative could be something like this -- the scrollbars are auto-hidden by default and appear when hovering, however, if a child element is hovered, also the parent's scrollbar shows. This means that more than 1 scrollbar could still show up at a time:
![auto-hide](https://user-images.githubusercontent.com/3294597/88040235-e21f3000-cb48-11ea-85d1-2533a4035694.gif)

Thoughts?

(Note: @barklund / @swissspidy This would mean using `hasObtrusiveScrollbars` everywhere where we need scrolling, perhaps that's not ideal, see [the test PR](https://github.com/google/web-stories-wp/pull/3326))

Reply:
@miina These 2 solutions are not better than what we currently have (ie visible scrollbar). Having a child scrollbar looks weird to me.

Reply:
I'm super confused by this entire thread.

AFAIK, with the use of `hasObstrusiveScrollbars`, and as clearly outlined in [this tutorial](https://www.filamentgroup.com/lab/scrollbars/), we can fix *precisely* the mentioned issue. We can have:

1. a scrollbar that is styled the way we want to
2. a scrollbar that auto-hides 

The downside, as is correctly pointed out, is that the auto-hiding scrollbar is the *OS level* scrollbar which cannot be styled. Thus, if @samitron7's requirement is proper styling for the scrollbar no matter what, we have two choices:

1. Compromise on styling, but do auto-hide (note: the always-visible scrollbar would still be styled)
2. Style the scrollbars, but make them thin and always visible (roughly what we already have)

At this point, I pretty much just need @samitron7 to vote on one vs two.

Reply:
Let's waste any more cycle on this and just go with #2 (Style the scrollbars, but make them thin and always visible (roughly what we already have). All the alternatives are not better.

Reply:
@samitron7 https://github.com/google/web-stories-wp/issues/3363 has been added to enhance the style of scrollbars in favor of auto-hiding -- moved it to UX Needed column for defining the final style.